source/docs/publish/include.md

```
@@ -34,13 +34,13 @@ When a PR for this is merged, it will show up in HACS after the first scheduled
 ## CI Checks
 
 When submitting a repository for the default repository a suite of CI checks will be run to ensure that the repository follows the requirements.
-All checks needs to pass for the repository to be included (unless something else are agreed on).
+All checks need to pass for the repository to be included (unless something else is agreed on).
 
 ### Check Brands
 
 _Only applies to integrations._
 
-This checks that the repository are added to https://github.com/home-assistant/brands
+This checks that the repository is added to https://github.com/home-assistant/brands
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)